bokeh/charts/_builder.py

```diff
@@ -76,8 +81,26 @@ class Builder(HasProps):
 
     x_range = Instance(Range)
     y_range = Instance(Range)
+    y_names = Seq(String)
+    x_names = Seq(String)
```

Comment:
Does every single builder always expect to use this pattern? If not, should probably make a builder subclass that has this functionality and builders that do support `x_names` and `y_names` can subclass from that instead.